deploy/kubernetes/helm/che/values.yaml

```diff
@@ -32,6 +32,8 @@ global:
   #  This value can be passed if custom Oidc provider is used, and there is no need to deploy keycloak in multiuser mode
   #  default (if empty) is true
   #cheDedicatedKeycloak: false
+  # cheSystemAdmin cannot contain email address yet
```

Comment:
It can contain now, right?

Reply:
no, i wrote it based on the latest info you gave me... as I haven't tested it now after pulling the changes from the server...

but based on my latest extensive tests on email address (before pulling the changes as of now), it doesn't work